common/presenters/single-requests-to-table-component.test.js

```diff
@@ -179,7 +189,7 @@ describe('#singleRequestsToTableComponent()', function () {
 
   describe('its behaviour', function () {
     beforeEach(function () {
-      output = presenter(mockMoves)
+      output = presenter(mockMoves)(mockMoves)
```

Comment:
This looks weird, but I'm not sure what the alternative is.

The first invocation needs the data to determine the columns that are shown. This would create an empty table if used.
The second invocation uses that table, with the provided data to provide a normal table view.

Reply:
Could you not call it with no args for the first call for the purpose of this test?

```javascript
output = presenter()(mockMoves)
```

Reply:
I'll add some tests for variants of columns - I'm not sure if I can meaningfully pass through empty column data, but I can at least test the dynamic columns feature.

Reply:
I've added tests for the various permutations of the columns - I still have the `(mockMoves)(mockMoves)` but I also have ones with `()(mockMoves)`